documents/generated/asciidoc/index.adoc

```diff
@@ -0,0 +1,378 @@
+= CSAF Validator Service
+team@openapitools.org
```

Comment:
This looks like a default... Should we override that? Or is it from the generator?
If so,I  guess that should be the address of the project... e.g. secvisogram[at]bsi.bund.de

Reply:
Yes, this should probably be a specific credits part. I'll insert the secvisogram[at]bsi.bund.de email here if that is okay.